[2022-08-17 01:23:06,764] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.link_dag scheduled__2022-08-16T00:00:00+00:00 [queued]>
[2022-08-17 01:23:06,777] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.link_dag scheduled__2022-08-16T00:00:00+00:00 [queued]>
[2022-08-17 01:23:06,778] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-17 01:23:06,778] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-17 01:23:06,779] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-17 01:23:06,801] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): link_dag> on 2022-08-16 00:00:00+00:00
[2022-08-17 01:23:06,814] {standard_task_runner.py:52} INFO - Started process 5344 to run task
[2022-08-17 01:23:06,826] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'csv_dag', 'link_dag', 'scheduled__2022-08-16T00:00:00+00:00', '--job-id', '202', '--raw', '--subdir', 'DAGS_FOLDER/testDag.py', '--cfg-path', '/tmp/tmphp8886_8', '--error-file', '/tmp/tmpv2q1qjzu']
[2022-08-17 01:23:06,837] {standard_task_runner.py:80} INFO - Job 202: Subtask link_dag
[2022-08-17 01:23:06,979] {task_command.py:371} INFO - Running <TaskInstance: csv_dag.link_dag scheduled__2022-08-16T00:00:00+00:00 [running]> on host 36e1df4dd6ce
[2022-08-17 01:23:07,085] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=csv_dag
AIRFLOW_CTX_TASK_ID=link_dag
AIRFLOW_CTX_EXECUTION_DATE=2022-08-16T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-16T00:00:00+00:00
[2022-08-17 01:23:47,476] {python.py:173} INFO - Done. Returned value was: None
[2022-08-17 01:23:47,490] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=csv_dag, task_id=link_dag, execution_date=20220816T000000, start_date=20220817T012306, end_date=20220817T012347
[2022-08-17 01:23:47,546] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-17 01:23:47,586] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
